src/refresh-indicator.jsx

```diff
@@ -24,7 +24,7 @@ const RefreshIndicator = React.createClass({
 
     /**
      * Override the theme's color of the indicator while
-     * it's status is "loading" or it's percentage is 100.
+     * it's status is "loading".
```

Comment:
Why did you remove ` or it's percentage is 100`.
Isn't this wording supposed to be the negation of the wording for the `color` property?